validator/client/runner.go

```diff
@@ -172,3 +184,35 @@ func handleAssignmentError(err error, slot uint64) {
 		log.WithField("error", err).Error("Failed to update assignments")
 	}
 }
+
+func (v *validator) handleAccountsChanged(ctx context.Context, accountsChangedChan chan struct{}) {
+	validatingPubKeysChan := make(chan [][48]byte, 1)
```

Comment:
We should defer close this channel

Reply:
Thanks, I will also close the channel in `service.recheckValidatingKeysBucket`.